Functions/Gherkin.ps1

```diff
@@ -576,72 +628,74 @@ function Invoke-GherkinScenario {
     Exit-MockScope
 }
 
-function Find-GherkinStep {
```

Comment:
I meant to completely remove this--not just comment it out. I guess I should first solicit comments on the removal of this unused function--because perhaps it's there for a reason? Depending on the responses I get, I'll either drop this commit or rebase the _actual_ removal of the function.